app/models/ability.rb

```diff
@@ -29,6 +29,6 @@ def default_abilities(user)
 
     can :read, Community
     can :read, Index, hidden: false
-    can :hide, Index if user.index_admin?
+    can :hide, Index, published?: true if user.index_admin?
```

Comment:
Don't think you want to do this, as those can get injected back into sql queries.

```
a = Ability.new(User.last)
Index.accessible_by(a, :hide).count
   (1.0ms)  SELECT COUNT(*) FROM "indices" WHERE "indices"."published?" = $1  [["published?", true]]
ActiveRecord::StatementInvalid: PG::UndefinedColumn: ERROR:  column indices.published? does not exist
LINE 1: SELECT COUNT(*) FROM "indices" WHERE "indices"."published?" ...
                                             ^
HINT:  Perhaps you meant to reference the column "indices.published_at".
: SELECT COUNT(*) FROM "indices" WHERE "indices"."published?" = $1
from /home/vagrant/.gem/ruby/2.3.0/gems/activerecord-5.0.0.beta3/lib/active_record/connection_adapters/postgresql_adapter.rb:590:in `async_exec'
```